### PR TITLE
feature/rpm: Collected patches from Fedora for apt-rpm

### PR DESCRIPTION
**Pull request by Conan-Kudo**
This PR includes patches and modifications from Fedora's Git repo for the apt package.

The patches have been adapted into git patch form from: http://pkgs.fedoraproject.org/cgit/rpms/apt.git/tree/

### CODE REVIEW

File: buildlib/archtable [hunk 1]
```diff
@@ -13,10 +13,10 @@ sparc64	sparc
...
 m68k	m68k
 arm.*	arm
-ppc	powerpc
-powerpc	powerpc
-ppc64		powerpc64
-powerpc64	powerpc64
+ppc	ppc
+powerpc	ppc
+ppc64		ppc64
+powerpc64	ppc64
```
Comment (fsateler) [inline buildlib/archtable:20]: This file is gone since 0.7.21

Comment (Conan-Kudo) [inline buildlib/archtable:20]: I know, @julian-klode told me. However, I still want this stuff merged in so that we have the historical data.

**Comment by Conan-Kudo** [comment]
@julian-klode Any chance you'll pull this in anytime soon?

**Comment by Conan-Kudo** [comment]
Seems there's no interest in this...

**Pull request closed without merging**